resources/spreasheetToJSON.js

```diff
@@ -0,0 +1,304 @@
+// This is to be used as a parser to extract JSON from google spreadsheets
+// This code should be inserted as a Tool -> Script editor -> Code.gs
+
```

Comment:
I've added this script just in case we want to follow through on the idea of saving data in JSON files. This script would allow us to parse survey responses into json. This script can be added as the tool and ran as a test in the spreadsheet and it would do beautiful JSON for us. JSON is very easy to manage in js, so I think this is something we should  definitely use. Please let me know if you are up for it :) . As it is pretty obvious, I really love JSON, so I am pretty bias. And I also like the idea of separating html structure and data (but that's just me).